sentry_member_team_map_withoutSCIM.py

```diff
@@ -130,13 +162,28 @@ def update_team_reg(self, memberid, teamname):
             }
             data["userName"] = member.get('email')
             data['email'] = member.get('email')
-            newuser = sentry_cloud.create_team_member(data)
-            logger.info("Created new user (userName, role) in cloud: %s, %s" % (data["userName"], data['role']))
-            #get id of new user
-            newuser_id = newuser.get("id")
             
-            #update id in common dictionary
-            updated_ids_dict[member['email']] = newuser_id
+            logger.info(f'[INFO] - onprem user "{data["userName"]}" ("{data["role"]}" role) does not exist in cloud')
+
+            # Member creation via API is not permitted at this time; commenting out.
```

Comment:
Alternative to commenting this section out (because you can no longer create members via API, apparently), we could just delete it. But, wanted to bounce this idea off someone else on the team before removing entirely.

Reply:
@cstavitsky  sounds good to me